tests/unit/components/enhanced-table/enum-filter/EnumFilter.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import { vi } from 'vitest'
+import { render, fireEvent, screen } from '@testing-library/react'
+
+import EnumFilter from '~/components/enhanced-table/enum-filter/EnumFilter'
+
+const mockedProps = {
+  column: {
+    filterEnum: [
+      {
+        label: 'adminTable.active',
+        value: 'active'
+      }
+    ]
+  },
+  filter: '',
+  setFilter: vi.fn(),
+  clearFilter: vi.fn()
+}
+
+describe('EnumFilter test', () => {
+
+  it('should open menu after clicking on filter icon', () => {
+    const { rerender } = render(<EnumFilter { ...mockedProps } />) 

```

Comment:
move render to beforeEach

Reply:
> move render to beforeEach

I didn't use **_beforeEach_** because in one of my test cases, I need to change the "**_filter_**" prop. 
I can only do this if the component renders independently in every test case 

https://github.com/ita-social-projects/SpaceToStudy-Client/pull/509/files#:~:text=it(%27should%20display,%7D)

Reply:
> move render to beforeEach

fixed